packages/cli/src/commands/deploy/aws-providers/packing.js

```diff
@@ -0,0 +1,63 @@
+import path from 'path'
```

Comment:
is `packages/cli/src/commands/deploy/aws-providers/packing.js` a good home? not sure where this file should live.

Reply:
My preference: move packing to a separate folder and called in "packaging/nft.js"

Reply:
Now in `packages/cli/src/commands/deploy/packing/nft.js`.

Is that what you intended, or did you want it in `packages/cli/src/commands/deploy/aws-providers/packing/nft.js` or other?